src/number/n_GF.jl

```diff
@@ -384,10 +384,10 @@ system with given characteristic, degree and string. If this is not the
 desired behaviour, one can pass `false` for the optional `cached` parameter.
 """
 function FiniteField(p::Int, n::Int, S::String; cached=true)
-   n >= 16 || p >= 2^8 && throw(DomainError())
-   !Nemo.isprime(Nemo.fmpz(p)) && throw(DomainError())
-   n*log(p) >= 20*log(2) && throw(DomainError())
-   p^n >= 2^16 && throw(DomainError())
+   p >= 2^8 && throw(DomainError(p, "p must be < 256"))
```

Comment:
How about this (that aligns the test and the error which IMHO makes it easier to read). Alas, that's a purely subjective thought, feel free to ignore!
```suggestion
   p < 256 || throw(DomainError(p, "p must be < 256"))
```